core/src/main/java/org/arquillian/smart/testing/filter/TestVerifier.java

```diff
@@ -8,4 +8,8 @@
 public interface TestVerifier {
 
     boolean isTest(Path resource);
+
+    default boolean isCore(Path resource) {
```

Comment:
Well I prefer this way because the logic is encapsulated so anyone can use either one or another. I think it is better for reader read the affirmative case and not the negative.